src/Navbar.js

```diff
@@ -60,7 +62,11 @@ const Navbar = (props) => {
   ), cssModule);
 
   return (
-    <Tag {...attributes} className={classes} />
+    <Tag {...attributes} className={classes}>
+      <div className="container-fluid">
```

Comment:
I wonder if this should be a prop that handles this vs hardcoding?  E.g. like: https://github.com/bestguy/sveltestrap/blob/master/src/Navbar.d.ts#L7

Reply:
👍🏼  that looks good. And we can default to fluid